terraform/environments/electronic-monitoring-data/modules/landing_bucket_iam_user_access/main.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_iam_user" "supplier" {
+  name = "${var.local_bucket_prefix}-${var.data_feed}-${var.order_type}"
+  tags = merge(
+    var.local_tags,
+    { order_type = var.order_type },
+    { data_feed = var.data_feed }
+  )
+}
+
```

Comment:
```suggestion
resource "aws_iam_group" "supplier_group" {
  name = "supplier_group"
}

resource "aws_iam_group_policy" "supplier_data_access" {
  name   = "put-s3-${var.data_feed}-${var.order_type}-policy"
  group  = aws_iam_group.supplier_group.name
  policy = data.aws_iam_policy_document.supplier_data_access.json
}

resource "aws_iam_user_group_membership" "supplier_membership" {
  user = aws_iam_user.supplier.name
  groups = [aws_iam_group.supplier_group.name]
}
```

Reply:
Not sure we need a user group as each user has different permissions i.e. fms-general has access to the fms-general bucket, fms-specials the fms-specials bucket etc. So no crossover worth grouping?